### ISSUE TITLE
clustering.CBASS returns values on the transformed data

### ISSUE THREAD

**Issue by michaelweylandt** [DataSlingers/clustRviz#10]
I would expect this to be true: 

```
cbass_fit <- CBASS(presidential_speech)
all(clustering(cbass_fit, percent = 0)$cluster.mean.matrix == t(presidential_speech))
#> [1] FALSE
```

(Note that the `t()` shouldn't be there either as noted in #9)

**Comment by jjn13** [comment]
This test addressing this has been added at

https://github.com/DataSlingers/clustRviz/blob/master/tests/testthat/test_cbass_clustering.R#L109-L131

I will close this out unless we think this is sufficient.

**Comment by michaelweylandt** [comment]
Hmmm.... I think I don't love the behavior that those tests check for. 

I'm not sure when the centers of the transformed data would be preferred to the centers of the original data. I  (sort of) understand why we do the transformation internally, but it's easy enough to undo it. 

In #38, I've gone the way of always returning on the original scale, regardless of pre-processing options used.

What do you think?

**Comment by jjn13** [comment]
I think undoing the transformations and returning on the original scale is fine. I think the tests above will be applicable, but will have to be modified to account for the new output.

**Comment by michaelweylandt** [comment]
The work in #38 returns everything in the "original space" regardless of pre-processing / standardization choices. I think it's a good choice, but am waiting for your review. 

The plotting functions don't (yet) commit to always using the original space. I think we should go that way, for consistency.

**Comment by michaelweylandt** [comment]
Closed by #38
